less/forum.less

```diff
@@ -1030,23 +1030,24 @@ body .Post.CommentPost {
         display: none;
 }
 /*  thêm ảnh nền vào thẻ */
-.TagsPage-content .TagTiles>li.TagTile:nth-child(8) {
-	background-image: url("../../less/background/bg_kinhte.jpg");
-}
-
-.TagsPage-content .TagTiles>li.TagTile:nth-child(11) {
-	background-image: url("../../less/background/bg_phimhan.jpg");
-}
-
-.TagsPage-content .TagTiles>li.TagTile:nth-child(9) {
-	background-image: url("../../less/background/bg_lamdep.jpg");
-}
-
-.TagsPage-content .TagTiles>li.TagTile:nth-child(10) {
-	background-image: url("../../less/background/bg_phimanh.jpg");
-}
-
-.TagsPage-content .TagTiles>li.TagTile:nth-child(6) {
-	background-image: url("../../less/background/bg_doisong.jpg");
-	
+.TagsPage-content {
+	.TagTiles {
+		& > li.TagTile {
+			&:nth-child(6) {
+				background-image: url("../../less/background/bg_doisong.jpg");
+			}
+			&:nth-child(8) {
+				background-image: url("../../less/background/bg_kinhte.jpg");
+			}
+			&:nth-child(9) {
+				background-image: url("../../less/background/bg_lamdep.jpg");
+			}
+			&:nth-child(10) {
+				background-image: url("../../less/background/bg_phimanh.jpg");
+			}
+			&:nth-child(11) {
+				background-image: url("../../less/background/bg_phimhan.jpg");
+			}
```

Comment:
please rebase or merge master branch into this branch and put all of this under 1 `.TagsPage-content {} `